### PR TITLE
ci: Add tests against virtual ECU

### LINKED ISSUE #83: Wire up CI with virtual ECU
Depends on #82

### PR DESCRIPTION
**Pull request by rumpelsepp**

### CODE REVIEW

**Comment by rumpelsepp** [comment]
@fkglr Is this approach good and as intended?

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -38,3 +38,66 @@ jobs:
...
+      run: |
+        poetry run gallia scan-services --target "tcp-lines://127.0.0.1:20162" --no-dumpcap --sessions 1 2 --check-session
+
+    - name: Test scan-sessions
+      run: |
+        poetry run gallia scan-sessions --target "tcp-lines://127.0.0.1:20162" --no-dumpcap --depth 2
+
+    - name: Test scan-identifiers
+      run: |
+        poetry run gallia scan-identifiers --target "tcp-lines://127.0.0.1:20162" --no-dumpcap --start 0 --end 100
```
Comment (peckto) [inline .github/workflows/tests.yml]: I'm thinking about, if we should do this for a bunch of SIDs

Comment (rumpelsepp) [reply .github/workflows/tests.yml]: yeah. We should especially specify `--sid`. Currently the default is used, which is questionable.

**Comment by rumpelsepp** [comment]
@fkglr Feel free to finish this.

**Pull request merged**